sdk/vision/imageAnalysis/samples/typescript/keyBasedAuthentication.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import createImageAnalysisClient, { ImageAnalysisClient, ImageAnalysisResultOutput } from '@azure/imageAnalysis';
+import { AzureKeyCredential } from '@azure/core-auth';
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const endpoint: string = process.env['VISION_ENDPOINT'] || '<cognitive services endpoint>';
+const key: string = process.env['VISION_KEY'] || '<cognitive services key>';
+const credential = new AzureKeyCredential(key);
```

Comment:
Maybe we don't need this as a sample, I think showing the different credential types in the README should be enough